src/config/database.json

```diff
@@ -0,0 +1,19 @@
+{
+  "development": {
+    "username": "dannyramos",
```

Comment:
One thing we could consider is changing the usernames to a generic name like  "postgres". So that any developers that pull down this code don't have create a "dannyramos" or "rostammahabadi" user/superuser in their local postgres. That way it's consistent across test and production and also just more readable :)

Reply:
Ah great catch Michelle!!